Huawei/netflow-9.conf

```diff
@@ -1,12 +1,42 @@
+
+slot 1
```

Comment:
you can remove "slots" part as this is already referenced in this file 
https://github.com/kentik/config-snippets/blob/cmcgrath138-patch-1/Huawei/netflow-9-linecards.conf
You can add ipv6 statement in that file